mycompany-seeq-link-connector-myconnector/src/main/java/com/mycompany/seeq/link/connector/MyConnection.java

```diff
@@ -109,7 +137,7 @@ public void connect() {
 
         // Second, perform whatever I/O is necessary to establish a connection to your datasource. For example, you
         // might instantiate a JDBC connection object and connect to a SQL database.
-        this.datasourceSimulator = new DatasourceSimulator(this.connectionConfig.getTagCount(), signalPeriod);
+        this.datasourceSimulator = new DatasourceSimulator(signalPeriod);
 
         if (this.datasourceSimulator.connect()) {
```

Comment:
I think this code should move to connect(). We don't usually connect inside initialize()